recipes/fastquast/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "fastQuast" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 0a13fe73bf80050d62a3083a00a2fd04894ffc26d5dca5360a78299b217c39b5
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - fastQuast=fastQuast.fastQuast:main
+    - fastquast=fastQuast.fastQuast:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+  run_requirements:
+    - python=2.7
+    - python>=3.6
```

Comment:
not sure I've seen this `run_requirements` before, and it's simultaneously requesting two different python versions?